contributors/devel/sig-release/cherry-picks.md

```diff
@@ -91,10 +91,10 @@ patch release branches.
     [active](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#detailed-release-history-for-active-branches)
     release branches where the fix is applicable.
 
-  - Set the value of the environment variable `GITHUB_TOKEN` to your personal
-    access token to avoid an interactive prompt. If `GITHUB_TOKEN` is not set
-    you will be asked for your github password: provide the github personal
-    access token rather than your actual github password.
+  - If `GITHUB_TOKEN` is not set you will be asked for your github password:
+    provide the github personal access token rather than your actual github
```

Comment:
```suggestion
    provide the github [personal access token](https://github.com/settings/tokens) rather than your actual github
```

even better with the link